src/sentry/api/endpoints/organization_user_details.py

```diff
@@ -12,6 +12,11 @@ class OrganizationUserDetailsEndpoint(OrganizationEndpoint):
     permission_classes = (MemberPermission,)
 
     def get(self, request: Request, organization, user_id) -> Response:
+        try:
+            int(user_id)
+        except ValueError:
+            return Response({"detail": "user_id must be an integer"}, status=400)
```

Comment:
Generally we prefer to `raise ValidationError("Reason")` for this kind of thing, then DRF will handle marking it as a 400.